arm-insights/2016-03-01/swagger/insightsManagementClient_LogProfiles.json

```diff
@@ -0,0 +1,268 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "InsightsManagementClient",
+    "version": "2016-03-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/microsoft.insights/logprofiles/{logProfileName}": {
```

Comment:
@gucalder - One thing that came to my attention.
`"/subscriptions/{subscriptionId}/providers/microsoft.insights/logprofiles/{logProfileName}": `
- This path does not have a {resourceGroupName}. Is this correct ?
- Can resources be created without being in the resourceGroup ?
/cc @johanstenberg, @anudeepsharma